Formula/aws-session-manager-plugin.rb

```diff
@@ -17,18 +17,22 @@ def install
     if Hardware::CPU.intel?
       if Hardware::CPU.is_64_bit?
         url "https://s3.amazonaws.com/session-manager-downloads/plugin/#{version}/ubuntu_64bit/session-manager-plugin.deb"
-        sha256 "342602b05552268966bcac2ec7698447e1202eb8e3d2943066dd9954697604ef"
+        sha256 "27f01987ed2977480c1ff322e0889270a3b8df43097f71e2667827e745e8bbba"
       elsif Hardware::CPU.is_32_bit?
         url "https://s3.amazonaws.com/session-manager-downloads/plugin/#{version}/ubuntu_32bit/session-manager-plugin.deb"
+        sha256 "428e1f8b1114f7dcd2b33028dd2484b0089d1746463a3fec65c1ffebd1120d9e"
       end
+    elsif Hardware::CPU.arm?
+      url "https://s3.amazonaws.com/session-manager-downloads/plugin/#{version}/ubuntu_arm64/session-manager-plugin.deb"
+      sha256 "dddd35f810d6d4461d032f1ade48ab95dc10c06b7e5d15c533800d0a26293ec4"
+    end
 
       def install
         system "ar", "x", "session-manager-plugin.deb"
         system "tar", "xf", "data.tar.gz"
         bin.install "usr/local/sessionmanagerplugin/bin/session-manager-plugin"
         prefix.install_metafiles
       end
-    end
```

Comment:
I have no idea why I had this here and it really bugs me because removing it looks correct.